packages/client/stories/Button.stories.tsx

```diff
@@ -0,0 +1,154 @@
+import { Link, Button } from '@chakra-ui/react';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import React, { useState } from 'react';
+import {
+  ButtonPrimary,
+  ButtonSuccess,
+  ButtonInfo,
+  ButtonWarning,
+  ButtonDanger,
+  ButtonNeutral,
+} from '../styles/ui-components/Chakra-Button';
+
+const elementTypes = { Button, Link };
+
+export default {
+  title: 'Components/Buttons',
+  component: Button,
+  argTypes: {
+    as: {
+      options: Object.keys(elementTypes),
+      mapping: elementTypes,
+      control: {
+        type: 'select',
+        labels: {
+          Button: 'Button',
+          Link: 'Link',
+        },
+      },
+    },
+  },
+} as ComponentMeta<typeof Button>;
+
+/*
+ * Example Button story with React Hooks.
+ */
+export const Primary: ComponentStory<typeof ButtonPrimary> = (args) => {
+  const [value, setValue] = useState('Secondary');
+  const [isPrimary, setIsPrimary] = useState(false);
+
+  // Sets a click handler to change the label's value
+  const handleOnChange = () => {
+    if (!isPrimary) {
+      setValue('Primary');
+    } else {
+      setValue('Secondary');
+    }
+    setIsPrimary(!isPrimary);
+  };
```

Comment:
You can move this logic in a wrapper and reuse it for each button type or just remove it entirely I used it just to have an example story when setting storybook up.